projects/event-api.md

```diff
@@ -0,0 +1,90 @@
+### Description
+
+The Client / RUM Sig is looking to finalize, complete and resolve the outstanding issues / PR's that exist around the Event API / SDK and the Semantic Conventions that defines what an "event" is.
+
+* Defined, identifying an `event`)
+* Created (API / Sdk)
+* will be created from Describe the goals, objectives, and requirements for the project. This include the motivations for starting the project now, as opposed to later.
+
+### Project Board
+
+Once approved by TC, a project should be managed using a GitHub project board. This project board should be pre-populated with issues that cover all known deliverables, organized by timeline milestones.
+
+A [Technical Committee](https://github.com/open-telemetry/community/blob/main/community-members.md#technical-committee) (TC) member associated with the project can create the board, along with a new project-specific GitHub label to automatically associate issues and PRs with the project. The project lead and all other relevant project members should have edit access to the board.
+
+Once created, please link to the project board here.
+
+Preemptively created a board with the relevant and related issues
+https://github.com/orgs/open-telemetry/projects/65
+
+### Deliverables
+
+Resolve the outstanding Event Issues / PR's which will define
+* What is an event? What does an event look like (what attributes identify it as an event) so that vendors, backends can perform any necessary routing, translation or storage.
+* How are events transported (Logs Only, Logs / Span Event, etc)
+* The semantic conventions to follow for defining "known" events
+  * Reserved (known) prefix's (either as the domain or name prefix) (possible known prefixes: `otel.`, `k8s.`, `azure.`, `aws.`, `ecs.`, `browser.`, `mobile.`, etc) all other "unknown" domains should/can be considered as "custom" by venders / backends.
+  * How / where are these are defined in semantic conventions (.yml)
+  * Updating tooling to properly handle the conventions
+  * Note: This group is not responsible for definition the semantic conventions of All domain specific events, only "how" those events will be represented / transported as an "event".
+* Define when an event should be used vs a `Span` or `LogRecord`
```

Comment:
This seems to be missing a key deliverable about driving the event API to stability.

Here's how I would frame the deliverables for this working group:
- Refine / stabilize event data model describing what an event is, the semantic meaning of event fields, and how they are sent over the network.
- Pave the way for defining event semantic conventions, allowing their schemas to be described in YAML and incorporated into build-tools. 
- Define select event semantic conventions, establishing patterns for future event semantic conventions to follow.
- Refine / stabilize event API and any associated features in the SDK.